gems/aws-sdk-securityhub/lib/aws-sdk-securityhub/client.rb

```diff
@@ -5158,7 +5158,7 @@ def get_configuration_policy(params = {}, options = {})
     #   resp.to_h outputs the following:
     #   {
     #     association_status: "FAILED", 
-    #     association_status_message: "Configuration Policy a1b2c3d4-5678-90ab-cdef-EXAMPLE11111 couldn\u2019t be applied to account 111122223333 in us-east-1 Region. Retry your request.", 
+    #     association_status_message: "Configuration Policy a1b2c3d4-5678-90ab-cdef-EXAMPLE11111 couldn’t be applied to account 111122223333 in us-east-1 Region. Retry your request.", 
```

Comment:
Whats this change from? This doesn't seem like this file should be getting changed here.

Reply:
I ran rake build. I think it's a difference in kramdown or something? I reverted it manually, but I think internal versions are dated and should be updated.